applications/crossbar/src/modules/cb_conferences.erl

```diff
@@ -64,14 +64,25 @@ init() ->
 -spec allowed_methods() -> http_methods().
 -spec allowed_methods(path_token()) -> http_methods().
 -spec allowed_methods(path_token(), path_token()) -> http_methods().
-allowed_methods() ->
-    [?HTTP_GET, ?HTTP_PUT].
-allowed_methods(_) ->
-    [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
-allowed_methods(_, <<"details">>) ->
-    [?HTTP_GET];
-allowed_methods(_, <<"mute">>) ->
-    [?HTTP_POST].
+-spec allowed_methods(path_token(), path_token(), path_token()) -> http_methods().
+
+allowed_methods() -> [?HTTP_GET, ?HTTP_PUT].
+
+allowed_methods(_) -> [?HTTP_GET, ?HTTP_POST, ?HTTP_PATCH, ?HTTP_DELETE].
+
+allowed_methods(_, <<"details">>) -> [?HTTP_GET];
+
+allowed_methods(_, <<"lock">>) -> [?HTTP_POST];
```

Comment:
@k-anderson I think it's counterintuitive to denote unmute action as "delete mute", and here is why: common pattern for REST is: GET -- get object, POST -- update object, PUT/DELETE -- create/delete object. In face we are talking here about participant state "object" (which is created and persisted by server automatically, therefore no DELETE/PUT verbs should be used). We can introduce "mute" object of boolean type, and POST true/false values to it though.